addons/OPT/GPS/fn_isVehicleVisible.sqf

```diff
@@ -0,0 +1,16 @@
+#include "macros.hpp";
+
+
+params ["_vehicle"];
+
+if (isNull _vehicle) exitWith { false };
+
+// Don't show destroyed
+if (damage _vehicle == 1) exitWith { false };
+// Only show manned
+if (({alive _x} count crew _vehicle) == 0) exitWith { false };
+// Only show manned by own side
+if (side group (crew _vehicle) select 0 != side group CLibPlayer) exitWith { false};
+
+
+true
```

Comment:
Ginge auch als einzelnes statement, was in SQF sogar minimal performanter sein drüfte (glaube ich):
```
! isNull _vehicle 
&&  {
        damage _vehicle < 1 // don't show destroyed
        && {alive _x} count crew _vehicle > 0 // only show manned vehicles
        && side group (crew _vehicle) select 0 == side group CLibPlayer // only show vehicles manned by players of the player's side
    }
```